nin/frontend/app/scripts/controllers/bottom.js

```diff
@@ -72,6 +72,17 @@
         }
       });
 
+      commands.on('multiplyLoopLengthBy', function(amount) {
+        if ($scope.loopEnd === undefined || $scope.loopStart === undefined) {
+          return;
+        }
+
+        var clampedAmount = Math.max(0, amount),
+            loopLength = $scope.loopEnd - $scope.loopStart;
+
+        $scope.loopEnd = Math.max(1, $scope.loopStart + (loopLength * clampedAmount));
```

Comment:
This clamp may not do what you think it does! As it is written, loops cannot end on frame 0. You may have intended to stop loops from starting and ending of the same frame, i.e. a loop length of 0. In that case, you should use something like loopStart + max(1, loopLength * clampedAmount) instead.